app/controllers/settings/general.js

```diff
@@ -24,10 +25,19 @@ export default Controller.extend(SettingsSaveMixin, {
     _scratchFacebook: null,
     _scratchTwitter: null,
 
+    imgMimeTypes: 'image/gif,image/jpg,image/jpeg,image/png,image/svg+xml',
```

Comment:
These shouldn't need to be duplicated, they are the same as the defaults in `gh-image-uploader`.